web-ng/src/app/components/form-field-editor/form-field-editor.component.ts

```diff
@@ -0,0 +1,84 @@
+import {
+  Component,
+  OnInit,
+  Input,
+  Output,
+  EventEmitter,
+  OnChanges,
+} from '@angular/core';
+import { FormGroup, FormBuilder } from '@angular/forms';
+import { FieldType } from '../../shared/models/form/field.model';
+
+export interface FieldTypeOptionModel {
+  icon: string;
+  label: string;
+  type: FieldType;
+}
+
+@Component({
```

Comment:
Please add basic TSDoc to each class and non-obvious public method.

Reply:
done!